--- autotrace-0.31.1/autotrace-config.in.old	2002-11-08 19:18:31.000000000 +0100
+++ autotrace-0.31.1/autotrace-config.in	2010-11-29 14:03:54.772079456 +0100
@@ -205,7 +205,7 @@
 #dummy because this should always be selected
 
 af_cflags="$af_cflags -I@includedir@"
-af_libs="-L@libdir@ -lautotrace @LIBPNG_LDFLAGS@ @MAGICK_LDFLAGS@ @LIBSWF_LDFLAGS@ @LIBPSTOEDIT_LIBS@ $af_libs"
+af_libs="-L`pkg-config --variable=libdir autotrace` -lautotrace @LIBPNG_LDFLAGS@ @MAGICK_LDFLAGS@ @LIBSWF_LDFLAGS@ @LIBPSTOEDIT_LIBS@ $af_libs"
 
 
 
